core/cmd/ocr2vrf_configure_commands.go

```diff
@@ -424,6 +427,10 @@ func createDKGJob(lggr logger.Logger, app chainlink.Application, args dkgTemplat
 }
 
 func createOCR2VRFJob(lggr logger.Logger, app chainlink.Application, args ocr2vrfTemplateArgs) error {
+	var sendingKeysString = fmt.Sprintf(`"%s"`, args.sendingKeys[0])
+	for x := 1; x < len(args.sendingKeys); x++ {
+		sendingKeysString = fmt.Sprintf(`%s,"%s"`, sendingKeysString, args.sendingKeys[x])
+	}
```

Comment:
I believe this is just a `strings.Join`:

```suggestion
	strings.Join(args.sendingKeys, ",")
```

You can probably pass this `Join` call inline below (line 442)

Reply:
Yeah I think I overengineered this a bit, I got worried about the TOML parsing it exactly correctly.